site/content/docs/5.3/components/spinners.md

```diff
@@ -43,7 +43,7 @@ The border spinner uses `currentColor` for its `border-color`, meaning you can c
 <div class="spinner-border text-primary" role="status">
   <span class="visually-hidden">Loading...</span>
 </div>
-<div class="spinner-border text-secondary" role="status">
```

Comment:
This section is about how you can customize the spinners with text utilities. If it's removed here, it doesn't make that much sense. Either we keep `.text-secondary` (or `.text-black`), or we remove it IMO.

Same comment for the growing spinner corresponding example.

Reply:
There's now a remaining untrue sentence for the growing spinner: "Here it is in orange, along with a secondary supported variant.". You can probably only keep the "Here it is in orange.".

Reply:
Once this is done, it'll lokks pretty fine. Couldn't sport any issue on my side.

Reply:
sorry for forgetting it